ciml/trainer.py

```diff
@@ -372,9 +382,10 @@ def build_dataset(dataset, build_name, limit, sample_interval, features_regex,
 
     # Load available run ids for the build name
     runs = gather_results.load_run_uuids('.raw', name=build_name)
-    # Apply the limit
-    if limit > 0:
-        runs = np.array(runs[:limit])
+    # Apply the slice
+    slice_fn = lambda x: int(x.strip()) if x.strip() else None
+    slice_object = slice(*map(slice_fn, slicer.split(":")))
```

Comment:
Do we have a docstring or something for how this works? I've been sitting here staring at it and am having a hard time visualizing in my head how this works, or how you would use the slicer arg with this.

Reply:
Heh, ok I thought the help text would be enough, I will add some docstring.
Basically you can setup specify any slice like you would do in an array e.g.

10:-2:4 -> start at element 10 until 2 elements before the end with pace 4

Reply:
Ah ok, that's what I thought based on the variable names and the default value for slicer arg. It just wasn't clear to me from the code (I guess I could have just spun up the interpreter to test it) by looking at it.